gulpfile.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+// mostly copy-pasta from https://github.com/jsperf/benchmark.js-wrapper/blob/c84d86d3a6a15a94a1acb2b99ec24562adfa1d27/gulpfile.js
+
+const gulp = require('gulp');
+
+const addSrc = require('gulp-add-src');
+const concat = require('gulp-concat');
+const insert = require('gulp-insert');
+const remoteSrc = require('gulp-remote-src');
+const replace = require('gulp-replace');
+// const uglify = require('gulp-uglify');
```

Comment:
Is `uglify` and its other use meant to be commented out? Is that just leftover debug?